src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -645,6 +829,51 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
           BACKGROUND_IMG_TEMPLATES.includes(templateID) &&
           renderMapBackgroundInput(templateID)}
 
+        <QuestionInput
+          className="DetailsForm__hidden-component"
```

Comment:
`QuestionInput QuestionInput--hidden` class inside `<QuestionInput>` component depending on a `hidden?:boolean` prop. Too much?

Reply:
I don't think it's too much, and it's a nice idea. But BM might want something else tomorrow and I don't think we should invest too much time for extra things like this one, where it might get remove tomorrow.